index.html

```diff
@@ -219,18 +131,18 @@ <h1> About me</h1>
               </div>
               </div>
       </section> 
-      <section class="contact container">
+      <section class="contact container" id="contact">
         <p class="title"> I'm always <br> 
             interested in hearing about new projects,so if you'd
              like to chat please get in touch.</p>
-         <form action="https://formspree.io/f/mdojzzgj" method="post">
+         <form action="https://formspree.io/f/mdojzzgj" method="post" class="form">
              
           <input type="text" class="item item1" name="firstName"  placeholder="First Name" required   maxlength="30">
           <input type="text" class="item item2" name="lastName" placeholder="Last Name" required   maxlength="30">
             <input type="email" class="item" placeholder="Email address" name="email" required >
            
             <textarea  name="message" id="tetarea" class="item textarea" cols="20" rows="7" placeholder="Hello Mariane. Glad to see you |"   maxlength="500" required></textarea>
-           
+            <div class="form-error"></div>
```

Comment:
- [ ] Your "form-error" container is always appearing on the screen even before the user submits the form. Please, make it invisible and only appear if the form is not valid with its message inside.